### ISSUE TITLE
Add license to repository

### ISSUE THREAD

**Issue by basilfx** [blin00/os#1]
Hi,

I came across your repository while searching for a C implementation of the Fortuna PRNG.

I want to use your code for an open-source project (LGPLv2.1 licensed), but this repository does not have a license. Therefore I want to ask you if you want to add one.

Kind regards,
Bas

**Comment by blin00** [comment]
MIT license added.

**Comment by basilfx** [comment]
Hi,

Sorry for bringing up this very old issue.

Ten years ago I took your Fortuna PRNG as a basis for a version used in [RIOT-OS](https://github.com/RIOT-OS/RIOT/blob/master/sys/random/fortuna/fortuna.h). Compared to where I started, the work has deviated a lot.

I'd like to ask you if I can relicense my derivate as LGPLv2.1 (the default RIOT-OS license). This would simplify thing a bit within RIOT-OS. Of course, I will still mention your name and the original source.

Kind regards,
Bas
